ietf-network-inventory.yang

```diff
@@ -350,14 +350,22 @@ module ietf-network-inventory {
                 "RFC 6933: Entity MIB (Version 4) -
                           entPhysicalParentRelPos";
             }
-            container parent-component-ref {
+            leaf parent {
+              type leaf-ref {
+                path "../../component/component-id";
+                require-instance false;
+              }
               config false;
               description
-                "A placeholder for adding the reference to parent
-                 component(s): to further discuss whether to define
-                 a parent attribute as RFC 8348 or a
-                 parent-component-references container as in
-                 draft-ietf-ccamp-network-inventory-yang-02.";
+                "The identifier of the component that physically
+                contains this component.
+                If this leaf is not instantiated, it indicates that 
+                this component is not contained in any other 
+                component.";
+              reference
```

Comment:
In RFC 8348 there is some text to describe the case where a component has more than one parent (e.g., a multi-slot board) which I think it is worthwhile copying here as well:

```suggestion
                component.";

                In the event that a physical component is contained by
                more than one physical component (e.g., double-wide
                modules), this node contains the name of one of these
                components.  An implementation MUST use the same name
                every time this node is instantiated.";
              reference
```